sdk/openai/azure-ai-openai/README.md

```diff
@@ -188,7 +188,7 @@ IterableStream<Completions> completionsStream = client
 completionsStream
     .stream()
     // Remove .skip(1) when using Non-Azure OpenAI API
-    // Note: the first chat completions can be ignored when using Azure OpenAI service which is a known service bug.
+    // Note: the first chat completions can be ignored when using Azure OpenAI Service which is a known service bug.
```

Comment:
Can you also update [this README sample code snippet](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/openai/azure-ai-openai/src/samples/java/com/azure/ai/openai/impl/ReadmeSamples.java#L158) file, this section is generated based on the file.